src/vue/common/EmptyListPlaceholder.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <tbody class="empty-list-placeholder">
+    <tr
+      class="empty-list-placeholder__brief"
+    >
+      <td
+        :colspan="colspan"
+        class="empty-list-placeholder__cell--align-center"
+      >
+        {{ message }}
+      </td>
+    </tr>
+  </tbody>
+</template>
+
+<script>
+export default {
+  name: 'empty-list-placeholder',
```

Comment:
actually, empty tbody placeholder. rename the `name` and the file accordingly pls